uint16 pwm1
uint16 pwm2
uint16 pwm3
uint16 pwm4
bool flight